programs/mango-v4/src/state/mango_account.rs

```diff
@@ -1196,6 +1197,19 @@ impl<
         Ok(())
     }
 
+    pub fn find_first_unused_perp_position(&self) -> Option<&PerpPosition> {
```

Comment:
I think "unused" might be misunderstood as "inactive". Maybe `_active_unused_` even if it's verbose?